Cargo.toml

```diff
@@ -15,14 +15,19 @@ name = "bitcoin_hashes"
 path = "src/lib.rs"
 
 [features]
-default = [ "std" ]
+default = ["std"]
 std = []
+# The no-std feature enables core2 and the alloc feature adds the alloc crate to that.
+# You can still just disable std by disabling default features, without enabling these two.
+no-std = ["core2"]
```

Comment:
Do we need this as a feature? Users should be able to specify the `core2` feature without this.

Reply:
removed, although I'm not sure how to best communicate to the developer what are the main features they should be interested in.  I expanded the comment.

Reply:
I'm not sure how to communicate features *at all* - they aren't really mentioned in anything, and just using `Cargo.toml` doesn't really solve it. Ideally we'd put it in the top-line `lib.rs` docs.